### ISSUE TITLE
High latency and low FPS on Linux 

### ISSUE THREAD

**Issue by icheered** [alvr-org/ALVR#868]
# Please follow the general troubleshooting steps first:

- [X] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and completed successfully the setup wizard.
- [X] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/alvr-org/ALVR/wiki)

If your issue is about games not working, please open an issue for every game and list the game in the title.

### Bug reports:

On starting steamVR the latency is very high (100-200ms) and the framerate is very low (30fps)

#### Environement

CPU: i7 10700
GPU: RTX 3080
OS: Linux Manjaro (Kernel version 5.15)
ALVR Version: 16.0.0
SteamVR Version: Latest
![image](https://user-images.githubusercontent.com/37150873/143076864-2856375d-6b43-43ac-87b2-b1e3bcbf373b.png)

Router is plenty fast, Oculus air link works totally fine (on Windows).
Moonlight is working properly.

**Comment by JosNun** [comment]
On linux with Nvidia, currently only software encoding with the CPU is supported, so performance will be quite poor. It also doesn't provide us with some information that we need to get the frame lined up with the pose properly, so they tend to get out of sync, and result in jitter. 

TLDR: Unfortunately, ALVR isn't really usable on linux with Nvidia, at least for now. There is work being done for the Nvidia support, but as to when (or if) that releases is TBD.

**Comment by icheered** [comment]
Thanks for the response, sad to hear that it isn't (yet) supported.

**Comment by Thelvaen** [comment]
@icheered I believe the last release brings Nvenc and should allow good performance from now on.

**Comment by urbenlegend** [comment]
Just tested with 17.0.1 on Arch using the AUR package. I am getting similar results to OP even with NVENC (confirmed in ALVR log page).

**Comment by RaccShaccin** [comment]
Same as urbenlegend, high latency even with nvenc

**Comment by urbenlegend** [comment]
Things have improved a lot with version 18, but latency is still slightly higher than with Virtual Desktop on Windows, and the resolution seems to be a lot lower.

**Comment by RaccShaccin** [comment]
temporary fix for the resolution was to lower refresh rate and up resolution and bitrate? it didint change the overall latency much. but it made the image marginially more clear.

**Comment by bwskin** [comment]
Tried on Arch Linux, version 18 and nightly 19 using nvenc. Works properly only when resolution set to 50%. Everything above makes it unusable. On the same setup on windows and AirLink everything works perfect. Anything new about this issue?
